packages/react/src/components/Tag/OperationalTag.tsx

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes, { ReactNodeLike } from 'prop-types';
+import React, { MouseEventHandler } from 'react';
+import classNames from 'classnames';
+import setupGetInstanceId from '../../tools/setupGetInstanceId';
+import { usePrefix } from '../../internal/usePrefix';
+import { PolymorphicProps } from '../../types/common';
+import Tag, { SIZES } from './Tag';
+
+const getInstanceId = setupGetInstanceId();
+
+const TYPES = {
+  red: 'Red',
+  magenta: 'Magenta',
+  purple: 'Purple',
+  blue: 'Blue',
+  cyan: 'Cyan',
+  teal: 'Teal',
+  green: 'Green',
+  gray: 'Gray',
+  'cool-gray': 'Cool-Gray',
+  'warm-gray': 'Warm-Gray',
+};
+
```

Comment:
Can these be imported from `Tag`?

Reply:
If we import from Tag it will come the `outline` annd `high-contrast` also. That's why I created separate for OperationalTag

Reply:
Makes sense, thanks for clarifying!